src/lib/components/organisms/AboutTorrust.svelte

```diff
@@ -68,11 +68,11 @@
 				<div class="buttons">
 					<Button href="https://github.com/torrust/torrust-index-backend" color="secondary">
```

Comment:
Hi @grmbyrn the URL is also wrong:

Old: https://github.com/torrust/torrust-index-backend
New: https://github.com/torrust/torrust-index